server/lib/report_server/post_processing/job.ex

```diff
@@ -43,16 +49,43 @@ defmodule ReportServer.PostProcessing.Job do
               # the accumulator is nil on the first line
```

Comment:
outdated comment?

Reply:
Not outdated but definitely vague - I've updated it in https://github.com/concord-consortium/report-service/pull/363/commits/8a44ee9d41734780144fd034dea3eee1c21c4e29.